### ISSUE TITLE
[DeepLab] Handing the data set that only includes masked image instead of annotation file

### ISSUE THREAD

**Issue by wenouyang** [tensorflow/models#4018]
Hi, 

I am trying to use Deeplab V3 for my own image set, which happens to be the format like Kitti Road Data set,

The training set consists of a pair of images, raw image and masked image, which are shown as follows. At present, there has code for transforming the cityscape/voc data into tfrecord. How to transform the data set, shown as follows, into tfrecord? 

![um_000003](https://user-images.githubusercontent.com/5430158/38943062-15d9ae12-42f6-11e8-96a6-572e9cbdd2ab.png)

![um_lane_000003](https://user-images.githubusercontent.com/5430158/38943102-2ca0d9b8-42f6-11e8-8ae4-e1bdcadae6f4.png)

**Comment by bitfort** [comment]
I'd recommend starting with http://stackoverflow.com/questions/tagged/tensorflow to answer questions about getting started. If you run into any specific issues or missing features, please do feel open an issue here about that specific bug or feature. Thanks!

Victor
